website/docs/core-concepts/connecting-to-data-sources/connecting-to-databases.md

```diff
@@ -5,59 +5,75 @@ description: >-
 ---
 
 # Databases
- You can connect to, and run queries to read and write data directly from the
-  Appsmith editor.
 
-## Supported Databases
+Databases are an essential part of many applications and are used to store and manage data. They provide a structured way to organize, retrieve, and manipulate data, and are designed to be efficient and reliable. You can run queries to read and write data directly from the Appsmith editor. 
 
-* [Amazon S3 (also Upcloud, Digital Ocean Spaces, Wasabi, DreamObjects)](../../reference/datasources/querying-amazon-s3.md)
-* [ArangoDB](../../reference/datasources/querying-arango-db.md)
-* [DynamoDB](../../reference/datasources/querying-dynamodb.md)
-* [ElasticSearch](../../reference/datasources/querying-elasticsearch.md)
-* [Firestore](../../reference/datasources/querying-firestore.md)
-* [Google Sheets](../../reference/datasources/querying-google-sheets.md)
-* [MongoDB](../../reference/datasources/querying-mongodb/)
-* [MS SQL](../../reference/datasources/querying-mssql.md)
-* [MySQL](../../reference/datasources/querying-mysql.md)
-* [PostgreSQL](../../reference/datasources/querying-postgres.md)
-* [Redis](../../reference/datasources/querying-redis.md)
-* [Redshift](../../reference/datasources/querying-redshift.md)
-* [Snowflake](../../reference/datasources/querying-snowflake-db.md)
-* [SMTP](../../reference/datasources/using-smtp.md)
+### Connecting to a database
 
-## Security
+<VideoEmbed host="youtube" videoId="sJIxtXInV14" title="How to connect to a datasource" caption="How to connect to a datasource | Example"/>
```

Comment:
please include the Test datasource action in the video. and maybe move the video to the bottom after the instructions? If there is an intro you can think of before the video, then do add it as directly starting with a video looks a bit odd

Reply:
moved video after steps